com.qq.QQ.yml

```
@@ -0,0 +1,70 @@
+app-id: com.qq.QQ
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+command: qq %U
```

this is an invalid command.

---

thank you, just pushed another patchset and addressed all your comments.

---

this remain invalid. There is NO argument.
```suggestion
command: qq
```